subscribers/slack/worker/Processors/UserMessageProcessor.cs

```diff
@@ -37,6 +37,20 @@ internal class UserMessageProcessor : AbstractMessageProcessor {
                         _logger.LogDebug("Supplier not supported for {@AwsSnsMessage}.", awsSnsMessage);
                     }
                     break;
+                case "abr_failed":
+                    var definition2 = new {
+                        user = new {
+                            email_address = "",
+                            role = "",
+                            name = ""
+                        }
+                    };
+                    var message2 = JsonConvert.DeserializeAnonymousType(awsSnsMessage.Message, definition2);
```

Comment:
Is it possible to define this var as `message`?

Reply:
The build fails when I do this. Since, it is `already defined in this scope`. I can change the variable name so its not message2